src/include/OSL/oslconfig.h

```diff
@@ -131,4 +131,11 @@ using OIIO::string_view;
 #  define OSL_DEPRECATED(msg)
 #endif
 
+// Work around bug in GCC with mixed __attribute__ and alignas parsing
```

Comment:
Maybe leave a link to the gcc bug you found? (In the distant future maybe we'll be able to re-evaluate if its still needed)